src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.html

```diff
@@ -1 +1,34 @@
-<p>vendor-filter works!</p>
+<body style="background-color:rgb(227, 227, 227);">
```

Comment:
Put styling in the scss file not inline styling

Reply:
this was strictly for testing because i couldn't see the buttons on the white background